site/spi/Makefile-bits-arnold

```diff
@@ -161,6 +161,127 @@ ifeq ($(SP_OS), spinux1)
 
 endif  # endif spinux1
 
+## Rhel7 (current)
+ifeq ($(SP_OS), rhel7)
+    platform=rhel7
+    SPCOMP2_COMPILER=gcc48m64
+    INSTALL_SPCOMP2_LOCATION = /shots/spi/home/lib/SpComp2
+    # At SPI, we have two "flavors" of spinux.  One is based on Foresight, which
+    # uses a special libGL (below).  The other is based on Fedora which uses
+    # the standard libGL.  This attempts to detect which libGL to use.
+    SPINUX_GL_LIB = /usr/lib64/xorg.nvidia.3d/libGL.so
+    MY_CMAKE_FLAGS += $(if $(wildcard ${SPINUX_GL_LIB}), -DOPENGL_gl_LIBRARY=${SPINUX_GL_LIB})
+    ifeq (${OCIO_PATH},)
+        OCIO_PATH="${INSTALL_SPCOMP2_LOCATION}/OpenColorIO/${SP_OS}-${SPCOMP2_COMPILER}/v2"
+    endif
+    ifeq (${FIELD3D_HOME},)
+        FIELD3D_HOME="${INSTALL_SPCOMP2_LOCATION}/Field3D/${SP_OS}-${SPCOMP2_COMPILER}-boost155sp/v406"
+    endif
+
+    ## If not overridden, here is our preferred LLVM installation
+    ## (may be changed as new versions are rolled out to the facility)
+    ifeq (${LLVM_DIRECTORY},)
+        LLVM_DIRECTORY := /shots/spi/home/lib/arnold/spinux1/llvm_3.4.2
+    endif
+
+    # default compiler is clang, taken from the LLVM directory
+    ifeq (${COMPILER},)
+        MY_CMAKE_FLAGS += \
+           -DCMAKE_C_COMPILER=${LLVM_DIRECTORY}/bin/clang \
+           -DCMAKE_CXX_COMPILER=${LLVM_DIRECTORY}/bin/clang++
+    endif
+
+    # requesting 'clang' or 'gcc' (no version) means the first clang or
+    # gcc in your path
+    ifeq (${COMPILER},clang)
+        MY_CMAKE_FLAGS += -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
+    endif
+    ifeq (${COMPILER},gcc)
+        MY_CMAKE_FLAGS += -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++
+    endif
+
+    # A variety of tags can be used to try specific versions of gcc or
+    # clang from the site-specific places we have installed them.
+    ifeq (${COMPILER}, clang341)
+        MY_CMAKE_FLAGS += \
+           -DCMAKE_C_COMPILER=/shots/spi/home/lib/arnold/spinux1/llvm_3.4.1/bin/clang \
+           -DCMAKE_CXX_COMPILER=/shots/spi/home/lib/arnold/spinux1/llvm_3.4.1/bin/clang++
+    endif
+    ifeq (${COMPILER}, clang342)
+        MY_CMAKE_FLAGS += \
+           -DCMAKE_C_COMPILER=/shots/spi/home/lib/arnold/spinux1/llvm_3.4.2/bin/clang \
+           -DCMAKE_CXX_COMPILER=/shots/spi/home/lib/arnold/spinux1/llvm_3.4.2/bin/clang++
+    endif
+    ifeq (${COMPILER}, gcc463)
```

Comment:
Just picking this one as an example... but are there any of these gcc variants that we know we'll never need?